src/components/editor/editor-pane/tool-bar/utils/toolbarButtonUtils.test.ts

```diff
@@ -988,62 +1029,63 @@ describe('test addComment', () => {
 })
 describe('test addTable', () => {
   const { cursor, firstLine, multiline, multilineOffset } = buildRanges()
-  const editor = Mock.extend(baseEditor).with({
-    getCursor: () => cursor.from
-  })
+
   const textFirstLine = testContent.split('\n')[0]
-  const table = '| # 1  | # 2  | # 3  |\n| ---- | ---- | ---- |\n| Text | Text | Text |'
+  const table = '|  # 1 |  # 2 |  # 3 |\n| ---- | ---- | ---- |\n| Text | Text | Text |'
```

Comment:
Any reason for this change here?

Reply:
because I changed the way the addTable function works in the mean time, and I just updated the output to be expected.

Reply:
ok